src/api/Alert/types.ts

```diff
@@ -75,6 +75,7 @@ export interface Alert extends IBaseAlert, PersistedItem {
     partial: boolean;
   };
   nextRun?: string; // defined if enabled=true
+  openAndUnAssignedCasesCount:number
```

Comment:
npm run lint, please @maxime-allex

Reply:
WTF! 🤣

Reply:
@francescoinfante I know you are bored, but you should be skiing :)